src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * 
+ */
+package seedu.addressbook.data.person;
+
+/**
+ * @author Nat
+ *
+ */
+public class Contact {
+	
+	  protected String value;
+	  protected boolean isPrivate;
+	  
+	  @Override
```

Comment:
You could have added a super class constructor here as well. Think conceptually how to design the super class, instead of just extracting the common methods and variables. It might help to come up with a better design for the super class.